docs/kubectl.md

```diff
@@ -904,12 +905,11 @@ Usage:
       --cluster="": The name of the kubeconfig cluster to use
       --context="": The name of the kubeconfig context to use
       --dry-run=false: If true, only print the object that would be sent, don't actually do anything
-      --generator="run-container/v1": The name of the api generator that you want to use.  Default 'run-container-controller-v1'
+      --generator="run-container/v1": The name of the api generator that you want to use.  Default 'run-container-controller/v1'
   -h, --help=false: help for run-container
       --image="": The image for the container you wish to run.
       --insecure-skip-tls-verify=false: If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure.
       --kubeconfig="": Path to the kubeconfig file to use for CLI requests.
-  -l, --labels="": Labels to apply to the pod(s) created by this call to run.
```

Comment:
Did you intend to remove this? It's included in the other help dumps.

If we keep it, it should say `run-container` rather than just `run`.